api/.gitignore

```diff
@@ -62,7 +62,7 @@ coverage.xml
 # Django stuff:
 *.log
 !woeip/apps/air_quality/tests/data/gps.log
-!woeip/apps/air_quality/tests/data/random.log
+!woeip/apps/air_quality/tests/data/random
```

Comment:
I think we can remove this from `.gitignore` since it was added due to `*.log` files being git ignored by default.

Reply:
👍